test-harness/src/main/java/io/jenkins/plugins/casc/misc/RoundTripAbstractTest.java

```diff
@@ -106,6 +107,9 @@ public void roundTripTest() throws IOException {
             String jenkinsConf = getResourceContent(resourcePath);
             assertConfigViaWebUI(jenkinsConf);
 
+            //Validate Schema before application
+            assertThat(validateSchema(convertToJson(resourceContent)), empty());
```

Comment:
Looks fine we should test this change on plugins that use this test and only do it after the nested schema change is merged

Reply:
Sounds cool +1

Reply:
build is failing btw